packages/ui/src/components/molecules/Tag/styles.scss

```diff
@@ -0,0 +1,99 @@
+[data-fs-tag] {
+  // --------------------------------------------------------
+  // Design Tokens for Tag
+  // --------------------------------------------------------
+
+  // Default properties
+  --fs-tag-text-color                     : var(--fs-color-text);
+
+  // Icon
+  --fs-tag-icon-size                      : var(--fs-spacing-3);
+  --fs-tag-icon-stroke-width              : var(--fs-spacing-4);
+
+  position: relative;
+  display: flex;
+  align-items: center;
+
+  [data-fs-badge-wrapper] {
+    position: relative;
+    display: flex;
+    align-items: center;
+    padding-right: var(--fs-spacing-6);
+    line-height: initial;
+  }
+
+  [data-fs-tag-icon-button] {
```

Comment:
In order to button's tap size works properly, we need to add a `overflow: visible` here.

Reply:
hmm. I didn't get this one 👀 
do we still need it here?